home/templates/pages/about.html

```diff
@@ -150,6 +182,58 @@ <h2>Meet Our Team</h2>
         <p>We are a diverse team of professionals, dedicated to delivering top-notch security solutions tailored for your business.</p>
     </div>
 
+    <!-- news letter pop-up-->
+    <!-- Trigger Button to Show Modal (Optional) -->
+    <button id="trigger-modal" style="display:none;">Open Newsletter Popup</button>
+
+    <!-- The Modal -->
+    <div id="newsletter-modal" class="modal">
+        <div class="modal-content">
+            <span class="close-button">&times;</span>
+            <h2>Join Our Newsletter</h2>
+            <p>Stay updated with our latest projects and news.</p>
+            <input type="email" id="email-input" placeholder="Enter your email" required>
+            <br>
+            <button id="subscribe-button">Subscribe</button>
+        </div>
+    </div>

```

Comment:
styling on this modal seems to be messed up in dark mode
text is white on white and its squished
![image](https://github.com/user-attachments/assets/da5630a5-f3a9-4219-85fb-4c970087eaac)

Reply:
styling is still broken in darkmode as of the latest changes